Dockerfile.prod

```diff
@@ -0,0 +1,17 @@
+FROM gradle:8.10.2-jdk17-alpine AS build
+
+WORKDIR /app
+
+COPY . .
+
+RUN ./gradlew --no-daemon build -x test
+
+FROM eclipse-temurin:17-alpine AS runtime
+
+WORKDIR /app
+
+COPY --from=build /app/build/libs/ArgosFileManager-0.0.1-SNAPSHOT.jar ./
+
+EXPOSE 8080
```

Comment:
I see `Dockerfile.dev` is having the same port as this one. It'd be nice if you change one.

Reply:
it is not necessary since they are not going to be executed at the same time, then there would be no problem of port occupation, also I already consulted with the team and they gave their approval.